pkg/proxy/iptables/proxier_test.go

```diff
@@ -423,6 +427,19 @@ func NewFakeProxier(ipt utiliptables.Interface) *Proxier {
 	return p
 }
 
+type trivialHasher struct{}
+
+func (trivialHasher) Hash(input ...string) string {
+	out := ""
+	for i, in := range input {
+		if i > 0 {
+			out += "-"
+		}
+		out += in
```

Comment:
`out += strings.ToUpper(in)` ?

Reply:
Yeah, will cleanup format to make this easier to see